src/AzureAppConfigurationImpl.ts

```diff
@@ -105,3 +108,23 @@ export class AzureAppConfigurationImpl extends Map<string, unknown> implements A
         return headers;
     }
 }
+
+function validatedSelectors(selectors?: { keyFilter: string, labelFilter?: string }[]) {
+    if (!selectors || selectors.length === 0) {
+        // Default selector: key: *, label: \0
+        return [{ keyFilter: KeyFilter.Any, labelFilter: LabelFilter.Null }];
+    }
+    return selectors.map(selectorCandidate => {
+        const selector = { ...selectorCandidate };
+        if (!selector.keyFilter) {
+            throw new Error("Key filters cannot be empty.");

```

Comment:
```suggestion
            throw new Error("Key filter cannot be null or empty.");

```

Reply:
In fact the type definition `keyFilter: string` guards that it cannot be `null` (if strictly checked). But again it won't work if type check is disabled/skipped. 

So yes, I will commit you suggestion to make it more clear.